src/_includes/dart-tool.md

```diff
@@ -0,0 +1,38 @@
+{{site.alert.note}}
+  As of Flutter's 1.19.0 dev release, the `dart` command
```

Comment:
"is now surfaced for you" seems unnecessarily indirect. How about:

...release, the Flutter SDK contains the `dart` command alongside the `flutter` command,
so that you can more easily run Dart command-line programs.

Reply:
(assuming that's the reason for the `dart` command to live there)